api/bases/barbican.openstack.org_barbicanapis.yaml

```diff
@@ -35,6 +35,10 @@ spec:
           spec:
             description: BarbicanAPISpec defines the desired state of BarbicanAPI
             properties:
+              clientCertificate:
```

Comment:
Maybe we should consider grouping these into their own key.  e.g. 

    spec:
      ...
      properties:
        pkcs11:
          clientCertificate: $CERT_CONTENTS_HERE

Reply:
Got it.  $CERT_CONTENTS_HERE will receive the OpenShift secret name that actually stores the certificate.

This file was automatically generated by a pre-commit hook.  I'll check how this could be translated into the specification file.